docs/overview/introduction.rst

```diff
@@ -0,0 +1,21 @@
+What It Is
+==================
+
+*CERN Analysis Preservation* (CAP) is a service to capture, preserve and reuse analysis information in HEP (high energy physics).
+It is developed by the collaborative effort of information and computer scientists at CERN, in partnership with the four largest LHC collaborations. A pre-release version is available to the latter for limited use.
```

Comment:
small thing/thought: maybe there is a better word for "limited use". Or does it need further explanation?